components/script/dom/webglframebuffer.rs

```diff
@@ -436,7 +496,10 @@ impl WebGLFramebuffer {
         attachment: u32,
     ) -> Option<&DomRefCell<Option<WebGLFramebufferAttachment>>> {
         match attachment {
-            constants::COLOR_ATTACHMENT0 => Some(&self.color),
+            constants::COLOR_ATTACHMENT0..=constants::COLOR_ATTACHMENT15 => {
```

Comment:
This changes the behaviour of passing COLOR_ATTACHMENTN in webgl1 contexts, as far as I can tell.

Reply:
In WebGL1, the number of color attachments is hardcoded to 1, so the color vector will also have only one element, and asking for anything other that COLOR_ATTACHMENT0 will return None like before.